part_3/cache.spec.js

```diff
@@ -0,0 +1,20 @@
+import { cache } from './cache.js';
+
+describe('cache::', () => {
```

Comment:
Правилом хорошего тона является описание по точному названию функции/модуля/неймспейса:
```suggestion
describe('cache', () => {
```